migrations/versions/241461fb2314_.py

```diff
@@ -0,0 +1,56 @@
+"""empty message
+
+Revision ID: 241461fb2314
+Revises: 3fae31bbf999
+Create Date: 2022-08-03 19:41:17.900433
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '241461fb2314'
+down_revision = '3fae31bbf999'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('circulating_ogn_supply',
```

Comment:
As opposed to creating a new table with no historical data which I supposed will cause issues on the front-end, how about renaming the table from `circulating_supply` to `circulating_ogn_supply`?

Reply:
ah right, I’ll have a look